lib/Doctrine/Query.php

```diff
@@ -544,8 +544,8 @@ public function parseSelectField($field)
             $field          = $terms[1];
         } else {
             reset($this->_queryComponents);
-            $componentAlias = key($this->_queryComponents);
-            $fields         = $terms[0];
+            $componentAlias = (string) key($this->_queryComponents);
+            $field          = $terms[0];
```

Comment:
I noticed that `$fields` was unused here, shouldn't have the `s`.

Reply:
Was curious how this was working and if it would change anything significant... the docblock seems to indicate that this method isn't used though.

Reply:
As far as I can tell, it isn't used (at least in the Doctrine code base). The only place this function is called is in the function itself (line 560).